codegenerator/cli/templates/dynamic/codegen/src/db/Enums.res.hbs

```diff
@@ -21,18 +21,11 @@ module ContractType = {
     | @as("{{contract.name.capitalized}}") {{contract.name.capitalized}}
   {{/each}}
 
-  let schema = 
-    {{#if codegen_contracts.1}}
-    S.union([
+  let schema = S.enum([
```

Comment:
Is this a new feature in RescriptSchema?

Reply:
Yes